package.json

```diff
@@ -1162,6 +1168,12 @@
                 "category": "AWS",
                 "icon": "$(cloud-upload)"
             },
+            {
+                "command": "aws.s3.uploadFileToS3",
+                "title": "%AWS.command.s3.uploadFileToS3%",
```

Comment:
can we reuse the existing `aws.s3.uploadFile` command instead of declaring a new one?